autopeering/arrow/arrow.go

```diff
@@ -0,0 +1,110 @@
+package arrow
+
+import (
+	"crypto/md5"
+	"encoding/binary"
+	"github.com/iotaledger/hive.go/identity"
+	"math/rand"
```

Comment:
I use math package, because I need the ArRow values generator to generate the same sequence of values for the same input and AFAIK crypto does not allow setting seed value.